_posts/platform/deployment/buildpacks/2000-01-01-nginx.md

```diff
@@ -66,10 +66,12 @@ Scalingo applications depending on the path:
 ```nginx
 location /api/v1 {
   proxy_pass <%= ENV["API_V1_BACKEND"] %>;
+  proxy_redirect <%= ENV["API_V1_BACKEND"] %>/api/v1 https://<%= ENV["APP"] %>.<%= ENV["REGION_NAME"] %>.scalingo.io/api/v1;
```

Comment:
Wouldn't it even be better to do it like that: 

```
  proxy_redirect <%= ENV["API_V1_BACKEND"] %>/api/v1 /;
```

Reply:
Or even if i'm reading the doc correctly;
```
proxy_redirect default;
```

Reply:
So that if the user has a custom domain defined, it wont redirect to appname.region.scalingo.io.

I'm also wondering if that's really something that we should advise by default or if the better advise wouldn't be to set proxy_redirect to off and let the user application routing do those kind of descision.

Reply:
I tested this here: https://dashboard.scalingo.com/apps/osc-fr1/biniou-proxy/